packages/@ama-sdk/schematics/schematics/api-extension/templates/src/definitions/__name__ExampleReply.yaml

```diff
@@ -0,0 +1,20 @@
+swagger: '2.0'
+info:
+  version: 0.0.0
+  title: <%= name %>ExampleReply
+  description: Definition for the DAPI model <%= name %>ExampleReply
+paths: {}
+definitions:
+  <%= name %>ExampleReply:
+    type: object
+    allOf:
+      - $ref: '@dxapi-spec/core-<%= coreType %>#/definitions/CommonReply'
```

Comment:
Do we want to keep this ref in an example?

Reply:
same